packages/cli/src/spec/configSpec.ts

```diff
@@ -0,0 +1,38 @@
+/*
+ * Copyright 2019 Google Inc. All Rights Reserved.
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ */
+
+import {join} from 'path';
+import {homedir} from 'os';
+import {existsSync} from 'fs';
+import {loadOrCreateConfig} from '../lib/config';
+import * as mock from 'mock-fs';
+
+describe('config', () => {
```

Comment:
We are messing with the user's file system, so I do reckon we could probably do with a few more tests (sorry, I know I said something different offline, but I've had a bit more thought about it now). Primarily, we want to check that:

* If there's no old config, we don't touch anything.
* If there's only an old config in `./llama-pack`, we should move the config and remove `./llama-pack`.
* If there are other files in `./llama-pack`, we should not delete that directory after moving the config.
* If there exists both an old config and a new config, we shouldn't overwrite the new `bubblewrap-config.json` with the old one.

(Sorry all of these cases weren't present in the original bug.)